test/WebSites/HtmlGenerationWebSite/Views/HtmlGeneration_Home/Order.cshtml

```diff
@@ -52,6 +52,9 @@
                 <option value="Credit">Credit</option>
```

Comment:
This particular set of files is "special" in that `Order.cshtml` and `OrderUsingHtmlHelpers.cshtml` should generate almost the same HTML. So, if you must change functional tests, change that and its output files too.